grafana-plugin/playwright.config.ts

```diff
@@ -49,7 +49,7 @@ export default defineConfig({
    * to flaky tests.. let's allow 1 retry per test
    */
   retries: 1,
-  workers: 2,
+  workers: '50%', // 50% of logical CPU cores, e.g. for 16 CPU cores it will use 8 workers
```

Comment:
@joeyorlando seems our e2e tests are still fine even when used 8 workers on CI :D

Reply:
<img src="https://media3.giphy.com/media/DhstvI3zZ598Nb1rFf/giphy-downsized-medium.gif"/>

Reply:
Nope, seems too much, they become flaky in this case